libr/anal/fcn.c

```diff
@@ -164,12 +164,18 @@ static bool isInvalidMemory(RAnal *anal, const ut8 *buf, int len) {
 	return !memcmp (buf, "\xff\xff\xff\xff", R_MIN (len, 4));
 }
 
-static bool isSymbolNextInstruction(RAnal *anal, RAnalOp *op) {
-	r_return_val_if_fail (anal && op && anal->flb.get_at, false);
+static bool is_symbol_flag(const char *name) {
+	return strstr (name, "imp.")
+		|| strstr (name, "dbg.")
+		|| strstr (name, "sym.")
+		|| strstr (name, "entry")
+		|| strstr (name, "main");
```

Comment:
shouldn't this be `strncmp` because `main` and `entry` should be the beginning?

Reply:
also don't we have the symbols listed, this seems kind of janky?

Reply:
yeah correct, that may solve some false positives i guess.